docs/develop/localhost.md

```diff
@@ -1,58 +1,59 @@
 # Local model serving
 
+Even with Truss, deploying a model to test changes creates a loop that is too slow for development. So instead we use Docker to create a fast local iteration loop that closely matches the production environment.
```

Comment:
Do you mean `Even with Baseten`? Othewise I'm confused what `even with Truss` means here.